event-handlers/src/test/java/no/sikt/nva/nvi/events/batch/EventBasedBatchScanHandlerTest.java

```diff
@@ -80,16 +80,15 @@ class EventBasedBatchScanHandlerTest extends LocalDynamoTestSetup {
   private static final int PAGE_SIZE = 4;
   private EventBasedBatchScanHandler handler;
   private ByteArrayOutputStream output;
-  private Context context;
+  private static final Context CONTEXT = mock(Context.class);
```

Comment:
We should probably have done the job needed to make FakeContext work, but fair here.